applications/ContactStructuralMechanicsApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -179,29 +185,68 @@ void  AddCustomStrategiesToPython(pybind11::module& m)
     class_< DisplacementLagrangeMultiplierContactCriteriaType, typename DisplacementLagrangeMultiplierContactCriteriaType::Pointer,
             ConvergenceCriteriaType >
             (m, "DisplacementLagrangeMultiplierContactCriteria")
+            .def(init<>())
+            .def(init<Parameters>())
             .def(init< double, double, double, double >())
             .def(init< double, double, double, double, bool >())
```

Comment:
honestly i think that by rule if you specify a non default value for something with a default you need to specify them all (in c++) so i am a bit surprised this works ... anyway, no problem as long as it compiles

Reply:
No, because in this case the default is on the Parameters, so I can assign empty